packages/plugins/plugin-assistant/src/components/ChatStream/ChatStream.stories.tsx

```diff
@@ -97,23 +90,10 @@ export const Default: Story = {
 
 export const Streaming: Story = {
   args: {
+    perCharacterDelay: 10,
     content: doc,
-    options: {
-      autoScroll: true,
-      fadeIn: true,
-      cursor: true,
-    },
+    autoScroll: true,
+    fadeIn: true,
+    cursor: true,
   },
 };
-
-export const Components = () => {
```

Comment:
i think you keep removing this story, but why? it's useful to have a story that doesn't involve the streaming but tests all of the xml tags.